src/components/Header/AnnouncementSearchForm/ButtonWithDropdownSection/ButtonWithIcons.jsx

```diff
@@ -0,0 +1,34 @@
+import PropTypes from 'prop-types';
+import { StyledButtonSection, StyledIconWrapper } from './ButtonWithIcons.styled.js';
+
+const ButtonWithIcons = ({
+  text,
+  iconLeftSide,
+  iconRightSide,
+  className,
+  onClick,
+  isOpenDropDown,
+}) => {
+  return (
+    <StyledButtonSection className={className} onClick={onClick}>
+      <img src={iconLeftSide} alt={`icon-for-${text}`} />
+      {text}
+      {iconRightSide && (
+        <StyledIconWrapper $isOpenDropDown={isOpenDropDown}>
+          <img src={iconRightSide} alt={`icon-for-${text}`} />
+        </StyledIconWrapper>
+      )}
+    </StyledButtonSection>
+  );
+};
+
+ButtonWithIcons.propTypes = {
+  text: PropTypes.string.isRequired,
+  iconLeftSide: PropTypes.any.isRequired, // TODO нічого крім any не працює, пошукай інші варіанти
```

Comment:
these icons are not "any". They are "string". But you should import they as ReactComponent if it possible

Reply:
+ import as as ReactComponent 
and then use PropTypes.elementType,